src/ECharts/Internal/EChartSeries.hs

```diff
@@ -209,7 +209,10 @@ instance ToJSVal EChartSeries where
   toJSVal = toJSVal_generic (drop $ T.length "_eChartSeries_")
 
 toEChartSeries :: Some SeriesT -> JSM EChartSeries
-toEChartSeries (Some.This st) = def
+toEChartSeries sst = Some.withSomeM (pure sst) $ \st ->
```

Comment:
anything that prevents using `Some.withSome` instead?

Reply:
What about `Some.Some`? I think that would make these changes much easier.

Reply:
I originally thought to use withSomeM because of the JSM ECharSeries return type, but I guess that doesn't actually justify the use of withSomeM

I also didn't know that Some.Some existed as a pattern synonym! Good to know

Reply:
Yeah it's rather buried in the new haddocks for some reason. If I didn't remember us creating it before I might not have seen it!